test/cli/dts/index.test.ts

```diff
@@ -3,17 +3,35 @@ import path from 'path'
 import { createCliTest } from '../utils'
 
 describe('cli', () => {
-  it(`cli dts should work properly`, async () => {
+  it('should generate .d.ts file on JavaScript file with --dts flag', async () => {
     await createCliTest(
       {
         directory: __dirname,
-        args: ['./input.ts', '-o', 'dist/output.js'],
+        args: ['./input.js', '-o', 'dist/javascript.js'],
       },
       ({ code, distFile }) => {
         const typeFile = distFile.replace('.js', '.d.ts')
 
-        expect(path.basename(distFile)).toBe('output.js')
-        expect(path.basename(typeFile)).toBe('output.d.ts')
+        expect(path.basename(distFile)).toBe('javascript.js')
+        expect(path.basename(typeFile)).toBe('javascript.d.ts')
+        expect(fs.existsSync(distFile)).toBe(true)
+        expect(fs.existsSync(typeFile)).toBe(true)
+        expect(code).toBe(0)
+      },
+    )
+  })
+
+  it('should generate .d.ts file on TypeScript file', async () => {
```

Comment:
moved position since after running this test, a tsconfig is generated, so the js test also passes.